api/iframe/copier.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <style> 
+/* so button can go to edge */
+body {
+  margin: 0;
+  font-family: Sans-Serif;
+}
+/* common styles for button */
+button#copy{
+  box-sizing: border-box;
+  font: inherit;
+  font-size: 18px;
+  line-height: 2em;
+  cursor: pointer;
+  border-radius: 4px;
+  border: 2px solid transparent;
+  background: #555;
+  padding: 0 8px;
+  color: #fff;
+  display: block;
+}
+/* cursor/size for within iframe */
+button#copy {
+  cursor: copy;
+  width: 100vw;
+  height: 100vh;
+}
+/* custom focus effect (outline would be cropped) */
+button#copy:active,
+button#copy:focus {
+  outline:0;
+  border-color:#0afaca;
+}
+  </style>
+</head>
+<body>
+<button id="copy">Copy</button>
+<input type="text" id="_i" hidden />
+<script>
+/*
+  Decode the location from the hash, placing it in our hidden text element. To copy, we show the
+  input, focus it, and call `execCommand('copy')`, before hiding it again.
+  See: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard
+ */
+_i.value = window.decodeURIComponent(window.location.hash.substr(1));
```

Comment:
Move this into the onclick handler to allow dynamically changing the fragment.

Reply:
Done